api/tests/acceptance/application/session/session-controller-get-supervisor-kit-PDF_test.js

```diff
@@ -25,21 +25,17 @@ describe('Acceptance | Controller | session-controller-get-supervisor-kit-pdf',
       });
 
       sessionIdAllowed = databaseBuilder.factory.buildSession({ certificationCenterId }).id;
-      sessionIdNotAllowed = databaseBuilder.factory.buildSession({
-        certificationCenterId: otherCertificationCenterId,
-      }).id;
 
       await databaseBuilder.commit();
     });
 
     it('should respond with a 200 when session can be found', async function () {
       // when
-      const authHeader = generateValidRequestAuthorizationHeader(user.id);
-      const token = authHeader.replace('Bearer ', '');
       const options = {
         method: 'GET',
-        url: `/api/sessions/${sessionIdAllowed}/supervisor-kit?accessToken=${token}&lang=fr`,
+        url: `/api/sessions/${sessionIdAllowed}/supervisor-kit`,
```

Comment:
```suggestion
        url: `/api/sessions/${sessionIdAllowed}/supervisor-kit?lang=fr`,
```

Pour limiter la suppression d'un élément de test hors scope de cette PR.